src/effects/RealtimeEffectManager.cpp

```diff
@@ -161,8 +161,8 @@ void RealtimeEffectManager::ProcessStart()
 // This will be called in a different thread than the main GUI thread.
 //
 size_t RealtimeEffectManager::Process(Track *track,
-                                      float **buffers,
-                                      size_t numSamples)
+   float *const *buffers, float *const *scratch,
+   size_t numSamples)
```

Comment:
in my simple test (have one mono track, instantiate BassTreble eff, press play) ::Process will exit early at line 173 because `mSuspended `will be true.